br/tests/lightning_fail_fast_on_nonretry_err/run.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+set -eu
+
+run_sql 'DROP DATABASE IF EXISTS csv'
+
+out_file_name="$TEST_DIR/sql_res.$TEST_NAME.txt"
+export GO_FAILPOINTS="github.com/pingcap/tidb/br/pkg/lightning/backend/local/WriteToTiKVNotEnoughDiskSpace=return(true)"
+start_time=$(date +'%s')
+run_lightning --backend local > $out_file_name 2>&1 || true
+export GO_FAILPOINTS=""
+check_contains "The available disk of TiKV"
+used_time=$(($(date +'%s') - $start_time))
+echo "time used to run lightning ${used_time}s"
+# writeAndIngestByRanges will retry for at least 810s on retryable error
+if [ $used_time -ge 60 ]; then
+  echo "lightning failed to fail fast"
+  exit 1
+fi
+
+run_sql 'SELECT count(*) FROM csv.threads'
```

Comment:
If run_sql returns error, should exit here.  Otherwise the next `check_contains` will exit normally and the whole test will pass

Reply:
check https://github.com/pingcap/tidb/pull/33794#discussion_r849020070